### PR TITLE
Feature: Add feedback for empty attributes in concepts

### PR DESCRIPTION
**Pull request by haddadzineddine**
## Description
When selecting a specific language, it is possible that the `Preferred Name` and `prefLabel ` attributes may be empty. To inform the user that these attributes do not have a value in the selected language, we need to add feedback.

## Screenshots:

1. synonyms and definitions :
image

2. prefLabel : 
image

3. Preferred name : 
image

## Reviewers
@syphax-bouazzouni

### CODE REVIEW

**Comment by syphax-bouazzouni** [comment]
Can you add screenshots for this

**Comment by haddadzineddine** [comment]
Use case :  [INRAE Thesaurus](http://[localhost:3000/ontologies/INRAETHES/?p=classes&conceptid=http%3A%2F%2Fopendata.inrae.fr%2FthesaurusINRAE%2Fc_21977](http://localhost:3000/ontologies/INRAETHES/?p=classes&conceptid=http%3A%2F%2Fopendata.inrae.fr%2FthesaurusINRAE%2Fc_21977))


image

File: app/views/concepts/_details.html.haml [hunk 1]
```diff
@@ -8,16 +8,20 @@
...
-          %p= @concept.prefLabel({:use_html => true}).html_safe
+          - if @concept.prefLabel().nil?
+            %p= t('ontology_details.concept.no_preferred_name_for_selected_language')
+          - else
+            %p= @concept.prefLabel({:use_html => true}).html_safe
+
       - unless @concept.synonym.nil? || @concept.synonym.empty?
         %tr
-          %td{nowrap: ""} Synonyms
+          %td{nowrap: ""}= t('ontology_details.concept.synonyms')
```
Comment (syphax-bouazzouni) [inline app/views/concepts/_details.html.haml:24]: Do the same for synonyms and definitions

**Comment by haddadzineddine** [comment]
## Screenshots:


1. synonyms and definitions :
image

2. prefLabel : 
image

3. Preferred name : 
image

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
As discussed in our last meeting:

- Show the warning only if the preferred name is empty
- Do a better style and reuse the alert component

**Comment by syphax-bouazzouni** [comment]
We will need a screenshot when done

**Comment by haddadzineddine** [comment]
@Bilelkihal I need the alter component to close this PR.

**Comment by haddadzineddine** [comment]
image

**Comment by haddadzineddine** [comment]
image

**Comment by haddadzineddine** [comment]
Screenshot 2023-05-24 203356

**Comment by syphax-bouazzouni** [comment]
Added: 

- Add padding the concept details no prefLabel warning (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/231/commits/d9d463b995ced70b26da3b94b82c7e7804a7d44b)
- Make the concept details no prefLabel alert closable (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/231/commits/236d86f23c7486bba3477619eb07d11d49266702)
- In concepts treeview the ones with no prefLabel show only their id last part (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/231/commits/ae916ac23d7204e5e9bc65efc5c53653c84fca53)

**Pull request merged**